source/_components/rdw.markdown

```diff
@@ -0,0 +1,98 @@
+---
+layout: page
+title: "RDW Sensor"
+description: "Instructions on how to set up RDW sensors within Home Assistant."
+date: 2019-05-04
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: rdw.png
+ha_category:
+  - Car
+  - Sensor
+ha_release: 0.93
+ha_iot_class: Cloud Polling
+---
+
+The `rdw` platform allows you to integrate data from the [RDW](https://www.rdw.nl) (Netherlands Vehicle Authority). It will track the APK expiration date, the insurance state and notifications if the manufacturer of the car has issued a recall because of a serious safety problem. It is possible to add multiple cars with Dutch license plates to your Home Assistant configuration.
+
+## {% linkable_title Configuration %}
+
+To enable this sensor, add the following lines to your `configuration.yaml`:
+
+```yaml
+sensor:
+  - platform: rdw
```

Comment:
Put only required items in the first example.

Reply:
Is it allowed to provide two examples? A simple one with only the required items and a more complex one?

Reply:
Yes ofcourse!